cmd/runtimetest/main.go

```diff
@@ -493,6 +493,25 @@ func validateMaskedPaths(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateSeccomp(spec *rspec.Spec) error {
+	if spec.Linux == nil || spec.Linux.Seccomp == nil {
+		return nil
+	}
+	for _, sys := range spec.Linux.Seccomp.Syscalls {
+		if sys.Action == "SCMP_ACT_ERRON" {
+			for _, name := range sys.Names {
+				if name == "getcwd" {
+					_, err := os.Getwd()
+					if err == nil {
+						logrus.Warnf("when action is %v, return value should being passed to user space as the errno value without executing the system call", sys.Action)
```

Comment:
I think the warning is not suitable, that's not what we really want to tell users.
We want to tell them even seccomp  is configured but this runtime can not apply it or similar warning